### PR TITLE
Pkgdown family

### PR DESCRIPTION
**Pull request by olivroy**
Probably will add changes in a follow-up PR. But this is ready to merge if you want

THe index is easier to scan.

image

### CODE REVIEW

File: _pkgdown.yml [hunk 1]
```diff
@@ -70,28 +74,37 @@ reference:
...
   desc: >
     These functions are helpers to create an intermediate R6 class object to use
     with `wb_*` functions.
   contents:
-  - wb_dims
   - wb_color
   - wb_comment
+  - wb_data
+  - wb_dims
+  - wb_hyperlink
```
Comment (olivroy) [inline _pkgdown.yml:107]:
```suggestion
```

**Review by olivroy — COMMENTED** [review]
rm

**Comment by JanMarvin** [comment]
It looks like I broke the pkgdown CI 😞

**Comment by JanMarvin** [comment]
Oh, thanks!

**Comment by JanMarvin** [comment]
Unrelated to this PR: Not really sure where this is coming from: the chartsheet currently lists references to `wb_add_plot()` etc. But IIRC it can only take mschart objects and not images.

**Comment by olivroy** [comment]
probably me trying to understand and document things at the same time. will address in a follow-up

**Comment by JanMarvin** [comment]
Thanks! Maybe it even works, I haven't actually tried this. (Also merged this PR)

**Comment by olivroy** [comment]
thank you so much! hope you like the new looks ! 

Sorry about so much back and forth..  I am confident that this is a good approach.

**Comment by JanMarvin** [comment]
I was just scrolling over the pkgdown page. Looks gorgeous! 😍
Thank you so much for all this hard work!

**Pull request merged**